contrib/Dockerfile.test

```diff
@@ -6,7 +6,7 @@
 # 3.) $ make binaries install test
 #
 
-ARG GOLANG_VERSION=1.12
+ARG GOLANG_VERSION=1.13.4
```

Comment:
Same here, let's just do "1.13"

Reply:
Actually not sure about this one. It might be referring to the release version 1.13

Reply:
This looks fine too